Description: Adds missing configuration elements for the maven-bundle-plugin that should be inherited from the parent pom
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/pom.xml
+++ b/pom.xml
@@ -75,6 +75,7 @@
             <plugin>
               <groupId>org.apache.felix</groupId>
               <artifactId>maven-bundle-plugin</artifactId>
+              <extensions>true</extensions>
               <configuration>
 		<instructions combine.children="merge">
 		  <Automatic-Module-Name>com.fasterxml.classmate</Automatic-Module-Name>
